exchanges/huobi/huobi_test.go

```diff
@@ -2752,7 +2752,9 @@ func TestFormatFuturesPair(t *testing.T) {
 	if err != nil {
 		t.Fatal(err)
 	}
-	if r != availInstruments[0] {
-		t.Errorf("expected %s, got %s", availInstruments[0], r)
+
+	// Upper case 'BTC' not 'btc' the rest is a changing contract code.
```

Comment:
Please rewrite, I don't know what you mean. Update the error message to. It won't be clear what to do on failure